setup.cfg

```
@@ -55,8 +55,8 @@ packages = find_namespace:
 python_requires = >=3.7
 include_package_data = True
 install_requires =
-    metomi-rose==2.0rc1
-    cylc-flow==8.0rc1.*  # the .* permits dev versions
+    metomi-rose==2.0rc2
```

Ah wait, looks like it was only the cylc-flow version that needed updating
```suggestion
    metomi-rose==2.0rc1
```

---

metomi-rose will need updating too, just hasn't been bumped yet.